api/v1/localvolume_types.go

```diff
@@ -64,6 +64,9 @@ type StorageClassDevice struct {
 	// A list of device paths which would be chosen for local storage.
 	// For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]
 	DevicePaths []string `json:"devicePaths,omitempty"`
+	// Boolean flag to enforce wiping devices before first use
```

Comment:
Maybe be more explicit what we mean by "wiping devices", something like "wiping existing filesystem signatures from the device".